python/lsst/db/db.py

```diff
@@ -21,32 +21,19 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 
 """
-This module is a wrapper around MySQLdb. It contains a set of low level basic
-database utilities such as connecting to database. It caches connections, and
-handles database errors.
+This module wraps SQLAlchemy.
```

Comment:
This is a confusing statement. We do not wrap whole sqlalchemy here, only one method which instantiates engine.